Runner/Runner.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import print_function
+from azure.common.credentials import ServicePrincipalCredentials
+import azure.batch.batch_service_client as batch
+import traceback
+import datetime
+import os
+import sys
+import json
+import asyncio
+import JobManager
+import Utils
+import azure.storage.blob as azureblob
+import azure.batch.models as batchmodels
+import azext.batch as batch
+import argparse
+"""
+This python module is used for valiading the rendering templates by using the azure CLI. 
+This module will load the manifest file 'TestConfiguration' specified by the user and
+create pools and jobs based on this file.
+"""
+
+sys.path.append('.')
+sys.path.append('..')
+
+_BATCH_ACCOUNT_NAME = os.environ['PS_BATCH_ACCOUNT_NAME']
+_BATCH_ACCOUNT_KEY = os.environ['PS_BATCH_ACCOUNT_KEY']
+_BATCH_ACCOUNT_URL = os.environ['PS_BATCH_ACCOUNT_URL']
+_BATCH_ACCOUNT_SUB = os.environ['PS_BATCH_ACCOUNT_SUB']
+_STORAGE_ACCOUNT_NAME = os.environ['PS_STORAGE_ACCOUNT_NAME']
+_STORAGE_ACCOUNT_KEY = os.environ['PS_STORAGE_ACCOUNT_KEY']
+_SERVICE_PRINCIPAL_CREDENTIALS_CLIENT_ID = os.environ['PS_SERVICE_PRINCIPAL_CREDENTIALS_CLIENT_ID']
+_SERVICE_PRINCIPAL_CREDENTIALS_SECRET = os.environ['PS_SERVICE_PRINCIPAL_CREDENTIALS_SECRET']
+_SERVICE_PRINCIPAL_CREDENTIALS_TENANT = os.environ['PS_SERVICE_PRINCIPAL_CREDENTIALS_TENANT']
+_SERVICE_PRINCIPAL_CREDENTIALS_RESOUCE = os.environ['PS_SERVICE_PRINCIPAL_CREDENTIALS_RESOUCE']
+
+_timeout = 20
+_job_managers = []
+
+
+if __name__ == '__main__':
+    start_time = datetime.datetime.now().replace(microsecond=0)
+    print('Sample start: {}'.format(start_time))
+    print()
+    print("_BATCH_ACCOUNT_NAME", _BATCH_ACCOUNT_NAME)
+    print("_BATCH_ACCOUNT_KEY", _BATCH_ACCOUNT_KEY)
+    print("_BATCH_ACCOUNT_URL", _BATCH_ACCOUNT_URL)
+    print("_BATCH_ACCOUNT_SUB", _BATCH_ACCOUNT_SUB)
+    print("_STORAGE_ACCOUNT_NAME", _STORAGE_ACCOUNT_NAME)
+    print("_STORAGE_ACCOUNT_KEY", _STORAGE_ACCOUNT_KEY)
```

Comment:
shouldn't print the credentials

Reply:
you should reset those keys too now

Reply:
Have talked to Matt about this and he is going to put the credentials in KeyVault as storing account keys in environment variables is a security risk.

Reply:
that's a good idea I think VSTS has good integration with keyvault

Reply:
Yep that's the plan, I also need to sort out what account I can use.

Reply:
I'll add logging that will print out the account name and the storage account being used, that's all

Reply:
Yep I'm using keyvault  now